src/net/SDP/SDPMediaAnnouncement.cs

```diff
@@ -82,6 +86,7 @@ public class SDPMediaAnnouncement
         public bool IceEndOfCandidates;         // If ICE candidate trickling is being used this needs to be set if all candidates have been gathered.
         public string DtlsFingerprint;          // If DTLS handshake is being used this is the fingerprint or our DTLS certificate.
         public int MLineIndex = 0;
+        public string IP;
 

```

Comment:
Is `IP` needed? If it's intended to represent the `c=` line it's already being represented with the `SDPConnectionInformation` class.

Reply:
removed